src/audio-worklet/shim/spn-demo/exporter.mjs

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Is this the exact copy of the utility function? Or do we have any changes to be reviewed?

Reply:
Exact copy, pulled outside of the wrapping class

Reply:
ACK